nvflare/app_opt/pt/job_config/base_fed_job.py

```diff
@@ -16,28 +16,35 @@
 
 from torch import nn as nn
 
-from nvflare import FedJob
+from nvflare.app_common.abstract.model_locator import ModelLocator
+from nvflare.app_common.abstract.model_persistor import ModelPersistor
+from nvflare.app_common.tracking.tracker_types import ANALYTIC_EVENT_TYPE
 from nvflare.app_common.widgets.convert_to_fed_event import ConvertToFedEvent
 from nvflare.app_common.widgets.intime_model_selector import IntimeModelSelector
+from nvflare.app_common.widgets.streaming import AnalyticsReceiver
 from nvflare.app_common.widgets.validation_json_generator import ValidationJsonGenerator
 from nvflare.app_opt.pt.job_config.model import PTModel
 from nvflare.app_opt.tracking.tb.tb_receiver import TBAnalyticsReceiver
+from nvflare.job_config.common_components_job import CommonComponentsJob
 
 
-class BaseFedJob(FedJob):
+class BaseFedJob(CommonComponentsJob):
```

Comment:
Can we simply combine CommonComponentsJob into BaseFedJob? BaseFedJob is already a "base" class, and it is not used directly by the users any way.

Reply:
thanks, merged them as suggested.